Cloud Application Manager/Deploying Anywhere/using-azure.md

```diff
@@ -20,12 +31,6 @@ Microsoft's Classic Azure hosts applications in virtual machines and in cloud se
 **Note:** If you’re running Cloud Application Manager as an appliance in your private datacenter, you can deploy only to the virtual machine role.
 To learn about Azure hosting services, see the [Azure help](https://azure.microsoft.com/en-us/documentation/articles/choose-web-site-cloud-service-vm/).
 
-**In this article:**
-* Before You Begin
-* Benefits of Deploying to Azure Using Cloud Application Manager
-* Registering Your Azure Subscription in Cloud Application Manager
-* Deploying in Azure
-
 ### Before You Begin
 
```

Comment:
In the **Focus on applications** section, the last sentence reads:

> For example, when you delete an instance that you no longer need, we automatically delete the availability set, virtual machine instances, storage account and cloud service, associated with the instance.dsfsdsd

The "dsfsdsd" at the end is a typo, I'm assuming?

Reply:
Yeah, good catch! Fixed.